applications/StructuralMechanicsApplication/tests/cpp_tests/test_total_lagrangian_mixed_volumetric_strain_element.cpp

```diff
@@ -144,6 +144,11 @@ namespace Kratos::Testing
      */
     KRATOS_TEST_CASE_IN_SUITE(TotalLagrangianMixedVolumetricStrainElementBonetPatch, KratosStructuralMechanicsFastSuite)
     {
+        // Skip the test if the constitutive law is not available (i.e. the ConstitutiveLawsApplication is not compiled)
+        if (!KratosComponents<ConstitutiveLaw>::Has("HyperElasticPlaneStrain2DLaw")) {
```

Comment:
perhaps use a variable for the name of the law, just in case it changes in the future

Reply:
Done.